examples/jira-scan-workflow/versions/0.0.1/docs/jira-sonar-template.md

```diff
@@ -0,0 +1,147 @@
+# jira-sonar-template
+
+## Summary
+This Workflow Template is used to create a CICD pipeline that clones both a source repository and deployment repository, builds an image, runs tests, scans and upgrades the image, and conduct a canary rollout all while creating and updating a jira during each step. 
+
+## Inputs/Outputs
+
+### Inputs
+* DOCKER_CONFIG_SECRET (required) - The k8s secret name from type docker-registry with all registries credentials you need to pull from or push to. defaults secret name `docker-config` . https://codefresh.io/csdp-docs/docs/getting-started/quick-start/create-ci-pipeline/#create-docker-registry-secret
+* REPO_URL (required) - The repository you want to clone that holds the source code (for example: https://github.com/codefresh-io/cli-v2)
+* GIT_TOKEN_SECRET (required) - The k8s secret name that holds the secret to your git token
+* GITHUB_TOKEN_SECRET_KEY (optional) - The k8s secret key that holds the secret to your git token. Default is 'token'
+* DEPLOYMENT_URL (required) - The repository you want to clone that holds deployment files (for example: https://github.com/codefresh-io/cli-v2)
+* JIRA_API_KEY (required) - The Kubernetes secret with the jira access key
+* JIRA_API_KEY_SECRET_KEY (optional) - The key in the Kubernetes secret with the Amazon access key. Default is 'api-key'
+* JIRA_BASE_URL (required) - Jira base url
+* JIRA_USERNAME (required) - The Kubernetes secret with the jira username
+* JIRA_USERNAME_SECRET_KEY (optional) - The key in the Kubernetes secret with the jira username. Default is 'username'
+* JIRA_ISSUE_SOURCE_FIELD (optional) - Jira issue ID or key source field
+* ISSUE_COMPONENTS (optional) - List of components using comma separated values: backend,database
+* ISSUE_CUSTOMFIELDS (optional) - Custom fields to pass to JIRA Issue Creation. Key=Value format.
+* ISSUE_DESCRIPTION (optional)- Jira issue description
+* ISSUE_PROJECT (optional) - Jira project key: necessary for issue creation
+* ISSUE_SUMMARY (optional) - Jira issue summary (main title)
+* ISSUE_TYPE (optional) - Jira issue type: Task, Bug, etc
+* COMMENT_BODY (optional) - Text to add to the comment
+* VERBOSE (optional) - Enable verbose logging by setting to true
+* REVISION (optional) - the revision to checkout. defaults to main
+* IMAGE_NAME (required) - The image name to give to the built image
+* DOCKERFILE (optional) - The path to your dockerfile. defaults to Dockerfile
+* CONTEXT (optional) - the context of file system that will be passed. defaults to .
+* TAG (optional) - The tag that will be given to the image. defaults to latest
+* CF_API_KEY (required) - The Kubernetes secret containing the Codefresh API key created by runtime
+* CF_API_KEY_SECRET_KEY (optional) - The key in the Kubernetes secret that has the Codefresh API key created by runtime. Default is 'token'
+* IMAGE_SHA (required) - The image sha that was imported into Codefresh to enrich
```

Comment:
this should not be a parameter, it is returned by image-report-info
one does not know the SHA image at the start of a build